src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -80,20 +80,21 @@ export default class DetailsHeader extends React.Component<Props> {
 
     const project = incident && incident.projects && incident.projects[0];
 
+    const alertBreadcrumbLabel = dateStarted ? (
+      <IncidentDate>{dateStarted}</IncidentDate>
+    ) : (
+      t('Alert details')
+    );
+
     return (
       <Header>
         <PageHeading>
-          <Breadcrumb>
-            <IncidentsLink to={`/organizations/${params.orgId}/alerts/`}>
-              {t('Alerts')}
-            </IncidentsLink>
-            {dateStarted && (
-              <React.Fragment>
-                <IconChevron direction="right" color={theme.gray1} size={space(1.5)} />
-                <IncidentDate>{dateStarted}</IncidentDate>
-              </React.Fragment>
-            )}
-          </Breadcrumb>
+          <AlertBreadcrumbs
+            crumbs={[
+              {label: t('Alerts'), to: `/organizations/${params.orgId}/alerts/`},
+              {label: alertBreadcrumbLabel},
```

Comment:
You should provide custom `key` here as the `label` was usually used for automatic `key` generation, but now the `label` got changed from string to node.

Reply:
Ah good point

Reply:
Fixed